install_files/ansible-base/securedrop-tails.yml

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ansible-playbook
+---
+# Configures SecureDrop dotfiles persistence on Admin and Journalist
+# Workstations. Should be run after the servers have been installed.
+- name: Configure Tails workstation.
+  hosts: localhost
+  connection: local
+  gather_facts: yes
+  roles:
+    - role: tails-config
+      tags: tails-config
+  tasks:
+    - name: Configuration complete.
+      debug:
+        msg: >-
+          Successfully configured Tor and set up desktop bookmarks for SecureDrop!
```

Comment:
so one issue i hit was that my desktop icons didnt show up right away -- i know thats not a new issue and cant find a specific fix online (gnome's refresh `alt`+`f2` --> `r`) didnt resolve it for me. rebooting did tho. so i dunno if we want to say `hey if your icons arent showing up on desktop, please reboot`

@conorsch responded in chat: `i noticed that too mike, you can quote my observation when you comment on github. my gut tells me there's some handler we can run to refresh the desktop, bu ti don't know what it is`